modules/cnf-programming-the-n3000-operator.adoc

```diff
@@ -0,0 +1,349 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="programming-the-openness-operator-for-intel-fpga-pac-n3000-operator_{context}"]
+= Programming the OpenNESS Operator for Intel® FPGA PAC N3000
+
+This section describes how to program the Intel® FPGA PAC N3000 with a vRAN 5G bitstream. This bitstream exposes the Single Root I/O Virtualization (SR-IOV) virtual function (VF) devices that can be used to accelerate the FEC in the vRAN workload.
+
+As a cluster administrator, you can install the OpenNESS Operator for Intel® FPGA PAC N3000 by using the {product-title} CLI or the web console.
+
+[id="programmimg-n3000-with-a-vran-bitstream_{context}"]
+== Programming the N3000 with a vRAN bitstream
+
+As a cluster administrator, you can program the Intel® FPGA PAC N3000 with a vRAN 5G bitstream. This bitstream exposes the Single Root I/O Virtualization (SR-IOV) virtual function (VF) devices that can be used to accelerate the Forward Error Correction (FEC) in the vRAN workload.
+
+[NOTE]
+====
+The role of Forward Error Correction (FEC) is to correct transmission errors, where certain bits in a message can be lost or garbled. Messages can be lost or garbled due to noise in the transmission media, interference, or low signal strength.
+Without FEC, a garbled message would have to be resent, adding to the network load and impacting both throughput and latency.
+====
+
+.Prerequisites
+
+* Intel FPGA PAC N3000 card
+* Performance Addon Operator with RT kernel configuration
+* Node or nodes installed with the N3000 Operator
++
+[NOTE]
+====
+All the commands run in the `vran-acceleration-operators` namespace.
+====
+
+.Procedure
+
+. Verify that the N3000 Operator is installed and that the pods are running:
```

Comment:
@kquinn1204 are certain permissions required to run this?

Reply:
I think is a cluster-admin the only one can apply this yamls

Reply:
I added "Log in as a user with cluster-admin privileges" to Prerequisites